fetch.bs

```diff
@@ -5685,7 +5686,14 @@ that RFC's normative processing requirements. [[RFC2397]]
 
  <li><p>Set <var>mimeTypeBytes</var> to the <a>isomorphic encode</a> of <var>mimeType</var>.
 
- <li><p>Return <var>mimeTypeBytes</var> and <var>body</var>.
+ <li><p>Let <var>dataURLStruct</var> be a <a>struct</a> consisting of a
```

Comment:
I was thinking "Let (mimeType, body) be the result of running the data URL processor". I guess that's not explicitly defined, but it should be.